codecov.yml

```diff
@@ -0,0 +1,46 @@
+# Enable coverage report message for diff on commit
+# Docs can be found here: https://docs.codecov.io/v4.3.0/docs/commit-status
+coverage:
+  status:
+    project:
+      tests:
+        target: 100%
+        # Overall coverage should never drop more then 0.5%
```

Comment:
comment is here but threshold is on line 20. also there the comment is only afterwards